packages/transactions/src/PaymentV2.ts

```diff
@@ -51,10 +51,9 @@ export default class PaymentV2 extends Transaction {
     this.signature = opts.signature
   }
 
-  serialize(): Uint8Array {
+  serialize(forSigning: boolean = false): Uint8Array {
     const BlockchainTxn = proto.helium.blockchain_txn
-
-    const paymentV2 = this.toProto()
+    const paymentV2 = this.toProto(forSigning=forSigning)
```

Comment:
This should just be `const paymentV2 = this.toProto(forSigning)` or `const paymentV2 = this.toProto(includeSignature)` if we change it

Reply:
I swore that I fixed this 😆  a consequence of primarily developing in Python